.pipelines/templates/install-oras.yaml

```diff
@@ -0,0 +1,12 @@
+parameters:
+  version: 0.8.1
```

Comment:
Not a blocker for this PR, but in a follow up this can be made configurable through pipeline variables